eventmesh-runtime/src/main/java/org/apache/eventmesh/runtime/util/RemotingHelper.java

```diff
@@ -50,6 +50,9 @@ public static String exceptionSimpleDesc(final Throwable e) {
 
     public static SocketAddress string2SocketAddress(final String addr) {
         int split = addr.lastIndexOf(":");
+        if (split == -1) {
```

Comment:
It's better to not do these kinds of changes in this PR, you just want to add UT right? Please revert the extra change.

Reply:
ok, already revert.